### DISCUSSION TITLE
聲紋辨識詐騙

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1737] [General]
AI技術逐漸氾濫，近期也聽說了有關詐騙集團透過電話來收集聲紋，並使用AI變聲來行使詐騙行為的案例。
我認為AI犯罪在未來只可能越來越普及，也因此大眾也許需要具備一些AI的基礎知識及了解才可能稍微做到防範的作用。我也在網上看到了一個對於AI變聲詐騙的應對方法，那就是和家人使用通關密語。對此我認為即使這可能是一個應對的辦法，但就連和家人打一通電話都變成了一個需要花心思提防的事，這不也是科技進步下的一個悲劇嗎？

**Comment by asd99263fg** [comment]
為了減少遇到詐騙，可以保持高度警惕，特別是在處理陌生人發送的訊息或電話時，避免輕易提供個人或財務資訊。定期更新和使用強密碼，並啟用雙重驗證以增加帳戶安全性。對於不明來歷的鏈接和附件，務必小心點擊，並確保所使用的網站是合法和安全的。此外，保持對最新詐騙手法的了解，並參考官方機構提供的防詐騙指南，以增強防範意識。
